plugin-dev/Source/Sentry/Private/Android/Java/SentryBridgeJava.java

```diff
@@ -89,6 +94,24 @@ public Double sample(SamplingContext samplingContext) {
 		});
 	}
 
+	private static void checkForUnrealException(SentryEvent event) {
```

Comment:
This does a whole lot more than just check for an Unreal exception as it returns nothing but modifies an event.

Reply:
Yeah, the naming could be better here - changed it to a more generic `preProcessEvent` yet I'm open to considering other options.